products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
+
+-   releaseCycle: "6"
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
+    latest: "6.4"
+    releaseDate: 2012-06-01
+    lts: 2022-06-30
+    latestReleaseDate: 2022-06-30
+    link: https://access.redhat.com/articles/3099701
+
+-   releaseCycle: "5"
+    support: 2013-11-30
+    eol: 2013-11-30
+    extendedSupport: 2019-11-30
+    latest: "5.2"
+    releaseDate: 2009-11-01
+    lts: 2019-11-30
+    latestReleaseDate: 2013-12-23
```

Comment:
Where can I double-check this date ?

Reply:
Working on it

Reply:
https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index

Reply:
https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/5

Reply:
https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/appe-release_notes_5.2-revision_history

Reply:
https://access.redhat.com/support/policy/updates/jboss_notes/
![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/0999efcf-aa70-4143-a3cc-3ca76c7091af)

Reply:
Could not find better than this : 
[2013-10-31](https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html-single/release_notes_5.2.0/index#appe-Release_Notes_5.2-Revision_History)

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/ed0e25a7-b6e7-4429-a840-4f58d6821875)